app/views/components/docs/download-link.yml

```diff
@@ -1,12 +1,19 @@
 name: Download link
 description: A link with a file download icon
+body: |
+  For usability the provided link text should indicate the file type and size of the download.
 accessibility_criteria: |
   The download link must:
 
   - accept focus
   - be focusable with a keyboard
   - be usable with a keyboard
   - indicate when it has focus
+  - change in appearance when touched (in the touch-down state)
+  - change in appearance when hovered
+  - be usable with touch
+  - be usable with speech
```

Comment:
I'm wondering if this is clear enough - obviously you mean with a screen reader, but it doesn't actually say that. Is there a WCAG page we can simply link to here to give a bit more information?